layers/build-krd/dracut/soci/soci-settled.sh

```diff
@@ -198,8 +215,8 @@ soci_udev_settled() {
         : > "${SOCI_FINISHED_MARK}"
         # if layer was 'tar' and there were no modules, then we can could unmount the iso fs.
         # if modules are mounted, or squashfs type layer, then this will fail.
-        out=$(umount "$dmp" 2>&1) ||
-            soci_debug "umount $dmp did did not succeed. Probably squashfs."
+        #out=$(umount "$dmp" 2>&1) ||
+            #soci_debug "umount $dmp did did not succeed. Probably squashfs."
```

Comment:
Do you still need this? do we not want to umount? if so drop it rather than comment.